data/mods/gen9ssb/conditions.ts

```diff
@@ -1227,6 +1239,90 @@ export const Conditions: {[k: string]: ModdedConditionData & {innateName?: strin
 			this.add('-end', pokemon, 'Attract', '[silent]');
 		},
 	},
+
+	gravity: {
```

Comment:
Which parts of gravity are modified? I see several methods that appear to be copy/pasted from base code. You are inheriting from the existing gravity condition already, so theres no need to include a method here unless you are overriding it to change how it works.

Reply:
Yep, most of this is indeed the same as vanilla gravity code, but inheriting doesn't seem to work in testing when I delete the rest of it. The onFieldEnd method at the end is the only one being overridden/modified.

Reply:
Yeah unfortunately Gravity is a move-based pseudoWeather so you have to kinda reimplement gravity to make any slight tweaks like this.